suite-native/device-manager/src/components/AddHiddenWalletButton.tsx

```diff
@@ -0,0 +1,27 @@
+import { useSetAtom } from 'jotai';
+
+import { Button } from '@suite-native/atoms';
+import { isPassphraseModalVisibleAtom } from '@suite-native/passphrase';
+import { useTranslate } from '@suite-native/intl';
+
+import { useDeviceManager } from '../hooks/useDeviceManager';
+
+export const AddHiddenWalletButton = () => {
+    const { translate } = useTranslate();
+
+    const setPassphraseModalVisibility = useSetAtom(isPassphraseModalVisibleAtom);
+
+    const { setIsDeviceManagerVisible } = useDeviceManager();
+
+    const handleAddHiddenWallet = () => {
+        setIsDeviceManagerVisible(false);
+        setPassphraseModalVisibility(true);
+        // await dispatch(createDeviceInstance({ device: instance }));
```

Comment:
leftover

Reply:
I left it on purpose, next task I'll be doing that so don't worry about it please.